notebooks/classification_examples/pytorch.ipynb

```diff
@@ -200,9 +163,131 @@
     "pycharm": {}
    },
    "source": [
-    "## Training\n",
+    "## Model\n",
     "\n",
-    "After we have done that, we can finally specify our experiment and run it. We will therfore use the already implemented `ClassificationNetworkBasePyTorch` which is basically a ResNet18:"
+    "After we have done that, we can specify our model: We will use a smaller version of a [VGG-Network](https://arxiv.org/pdf/1409.1556.pdf) in this case. We will use more convolutions to reduce the feature dimensionality and reduce the number of units in the linear layers to save up memory (and we only have to deal with 10 classes, not the 1000 imagenet classes)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from delira.models import AbstractPyTorchNetwork\n",
+    "import torch\n",
+    "\n",
+    "class Flatten(torch.nn.Module):\n",
+    "        \n",
+    "    def forward(self, x):\n",
+    "        return x.view(x.size(0), -1)\n",
+    "\n",
+    "class SmallVGGPyTorch(AbstractPyTorchNetwork):\n",
+    "    def __init__(self, in_channels, num_classes):\n",
+    "        super().__init__()\n",
+    "        \n",
+    "        self.model = torch.nn.Sequential(\n",
```

Comment:
Maybe we should use a simpler model so it is a bit easier for beginners?

Reply:
That's why I opted for VGG and not for any Resnet like model. What do you suggest?

Reply:
To demonstrate the general capability of training a NN with Delira we would only need a very small net. Some conv+pool and fc (around 3-4 layers) should be sufficient. Something like this https://www.tensorflow.org/tutorials/keras/basic_classification :)

Reply:
Okay, I'll change this